packages/editor/src/ui/DisplayModeToggle/index.tsx

```diff
@@ -42,14 +43,15 @@ export const DisplayModeToggle: React.SFC<{
   translations?: typeof defaultTranslations;
   stickyNess?: StickyNess;
 }> = ({ stickyNess, translations = defaultTranslations }) => {
+  const { t } = useUiTranslator();
   const actions = [
     // eslint-disable-next-line react/jsx-key
     { action: <UndoRedo labelRedo="redo" labelUndo="undo" /> },
-    { action: <ToggleEdit label={translations.edit} /> },
-    { action: <ToggleInsert label={translations.insert} /> },
-    { action: <ToggleLayout label={translations.layout} /> },
-    { action: <ToggleResize label={translations.resize} /> },
-    { action: <TogglePreview label={translations.preview} /> },
+    { action: <ToggleEdit label={t(translations.edit)} /> },
+    { action: <ToggleInsert label={t(translations.insert)} /> },
+    { action: <ToggleLayout label={t(translations.layout)} /> },
+    { action: <ToggleResize label={t(translations.resize)} /> },
+    { action: <TogglePreview label={t(translations.preview)} /> },
```

Comment:
this is one confusion that we have to discuss. Some stuff already has these translations. Should we keep them or should we migrate away?

Reply:
The existing translations object would require developers to provide it at every component as a prop, very performant and easy to use, but require a lot of work both for you and developers. If you agree with the current solution, I think removing `translations` prop is the next step.

Reply:
agreed!